pom.xml

```diff
@@ -602,6 +602,38 @@
                 <native-image.container-runtime>docker</native-image.container-runtime>
             </properties>
         </profile>
+
+        <profile>
+            <id>no-apache-snapshots</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
```

Comment:
Remote snapshots are evil because they make builds non-reproducible. I vote for having this the other way round: off by default, active only in a non-default profile.

Reply:
The apache snapshots have been enabled by default since long time ago and I have no problem to change the default behavior but I didn't want to change it while doing some experimentation because I do not remember if there is a reason or a job that need this set-up.

Reply:
I've now moved the snapshot repo to a dedicate profile